assets/src/scripts/components/map/nldiMapping.js

```diff
@@ -55,37 +55,34 @@ export const addNldi = function(map, legendControl, siteno) {
     };
 
     const addPointDataToMap = function(data, markerOptions) {
-        const pointLayer = geoJson(data, {
+        return geoJson(data, {
             onEachFeature: onEachPointFeatureAddPopUp,
             pointToLayer: function (feature, latlng) {
                 return circleMarker(latlng, markerOptions);
             }
         });
-        pointLayer.setZIndex(999);
-        map.addLayer(pointLayer);
     };
 
     const addLineDataToMap = function(data, style) {
-        const lineLayer = geoJson(data, {
+        return geoJson(data, {
             style: style
-            });
-        lineLayer.addTo(map);
+        });
     };
 
     const addNldiLinesToMap = function(endpointUrl, style) {
-        get(endpointUrl)
+        return get(endpointUrl)
```

Comment:
Consider changing the name of this function from addNldiLinesToMap to something like fetchNldiLinesLayer